package.json

```diff
@@ -73,7 +73,6 @@
     "redux-module-builder": "0.0.12-0",
     "redux-seamless-immutable": "^0.3.0",
     "redux-thunk": "^2.1.0",
-    "render-if": "^0.1.1",
```

Comment:
i just started using this library (render-if) in my PR!  is there a reason it was deleted besides not being used